modules/navier_stokes/test/tests/finite_volume/ins/boussinesq/tests

```diff
@@ -46,11 +46,13 @@
   [rho-error]
     type = RunException
     input = boussinesq-action.i
-    cli_args = "Modules/NavierStokesFV/density='x'"
+    cli_args = "Modules/NavierStokesFV/density='rho_error'"
     expect_err = 'rho: The density in the boussinesq term is not constant!'
     requirement = 'The system should throw an error if the density is not a constant functor in case of Boussinesq treatment.'
     valgrind = 'none'
     issues = '#19742'
+    # New physics cannot support this because we want properties forwarded from fluid properties to work
```

Comment:
I don't quite understand. So the Physics version is not going to take density as a functor?

Reply:
So how does Physics affect this? And does this test make sense to exist? Do we just need to improve `isConstant` somehow?

Reply:
>  Do we just need to improve isConstant somehow?

I dont see an easy way. Save for maybe evaluating it all over the mesh to check.

> So how does Physics affect this?

I dont want to limit incompressible flow to not using the FP properties and the GFFP. I ll give it another try to see if I can re-enable this.